CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+If you have an issue to report specific to the source code in this repository you can [open a GitHub issue](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange/issues). Before opening an issue please review the existing suggestions to see if your idea is already there. If already present, please comment on the existing issue instead of making a new one.
+
+If you have a general inquiry about Rearc's data services you can send an email to data@rearc.io. We would love to hear any suggestion, question or request you may have. 
+
+## What should I know before getting started?
+
+#### What Is AWS Data Exchange?
+> [AWS Data Exchange](https://aws.amazon.com/data-exchange/) is a data marketplace that makes it easy for AWS customers to securely find, subscribe to, and use third-party data in the cloud.
+
+#### Who is Rearc?
+[Rearc](https://www.rearc.io) is a data provider and one of the launch partners for AWS Data Exchange. Products published by Rearc on ADX can be found [here](https://aws.amazon.com/marketplace/seller-profile?id=a8a86da2-b2d1-4fae-992d-03494e90590b). On ADX we provide a pipeline to automate the (1) sourcing of data, (2) transformation of data to make it useful to subscribers, (3) creation of a dataset for public consumption, (4) automatic revisions of the dataset and, (5) publishing of the data product through ADX.
+
+#### What are Rearc's goals for ADX?
+We at Rearc are working tirelessly to lend greater accessibility to interesting and/or important datasets across various disciplines and sources. We realize the direct integration of the ADX, along with other AWS services, facilitates a convenient manner for our subscribers to consume data. For data providers we can supply an automation pipeline, leveraging the AWS platform, to ensure the ubiquity of your data for your consumers.
+
+#### What is Rearc's philosophy towards dataset formats?
+We try as much as possible to preserve the integrity of data we provide through ADX, and most of the time this means delivering datasets exactly as they were presented from their source. Sometimes we make minor alterations to datasets to provide wider usability for ADX subscribers (e.g. adjusting CSV files for SQL column naming conventions). For situations where we are unable to maintain the original data file format (often due to a request from the data consumer or a discrepancy in a source's delivery pipeline), we try to limit the extent of needed transformations as much as possible.
+
+#### What tools are you using throughout your ADX Products?
+- On our local devices we have chosen to use a combination of [Python 3](https://www.python.org), [Pip](https://pypi.org/project/pip/), [JQ](https://stedolan.github.io/jq/), [AWS CLI V2](https://aws.amazon.com/cli/) and other related developer tools. The specific tools utilized varies on a project-by-project basis. The Python portions of our source code can be adapted into another language with a supported AWS Lambda runtime.
```

Comment:
I would cut the last line about AWS supporting languages other than Python.

Reply:
Removed the line about supporting languages other than python